src/stores/currentUser.js

```diff
@@ -1,6 +1,8 @@
 import jwt_decode from "jwt-decode";
 import { defineStore } from 'pinia'
 import RouteError from '../errors/RouteError.js'
+import systemMessages from './systemMessages.js'
```

Comment:
A pinia convention is to give these functions a name starting with use. So in this case, it will be useSystemMessages. This way in the code later, you can do: useSystemMessages().addError(e)

Reply:
It's their convention, because it reflects the fact, that the current file is using the imported store.